src/main/java/org/opentripplanner/graph_builder/module/osm/WayPropertySet.java

```diff
@@ -88,26 +89,24 @@ public WayProperties getDataForWay(OSMWithTags way) {
     List<WayProperties> leftMixins = new ArrayList<>();
     List<WayProperties> rightMixins = new ArrayList<>();
     for (WayPropertyPicker picker : wayProperties) {
-      OSMSpecifier specifier = picker.specifier();
+      OsmSpecifier specifier = picker.specifier();
       WayProperties wayProperties = picker.properties();
-      P2<Integer> score = specifier.matchScores(way);
-      int leftScore = score.first;
-      int rightScore = score.second;
+      var score = specifier.matchScores(way);
       if (picker.safetyMixin()) {
-        if (leftScore > 0) {
+        if (score.left() > 0) {
```

Comment:
I wonder if there could be more descriptive words for these left/right? Isn't this the same back/forward stuff? I think we should use the same words to describe them everywhere.

Reply:
This actually refers to the left/right hand side of the street so I think the names are appropriate.

Reply:
Ok, in that case, these are fine.